packages/keeper/hardhat.config.js

```diff
@@ -1,9 +1,9 @@
 require('@nomiclabs/hardhat-ethers');
 
 module.exports = {
-  solidity: '0.8.4',
+  solidity: '0.8.9',
+  defaultNetwork: 'hardhat',
```

Comment:
defaultNetwork hardhat?

Reply:
Goig to remove the key, that's just the default value, so, there's no need to set it :)

Reply:
```suggestion
```